code42/py42
# Return py42 response on all writes

### Description of Change ###

- functions now return `Py42Response` objects as expected:
    - `sdk.detectionlists.update_user_notes()`
    - `sdk.detectionlists.add_user_risk_tags()`
    - `sdk.detectionlists.remove_user_risk_tags()`
    - `sdk.detectionlists.add_user_cloud_alias()`
    - `sdk.detectionlists.remove_user_cloud_alias()`
- `sdk.archive.get_all_org_cold_storage_archives()` now actually uses parameters `include_child_orgs`, `sort_key` and `sort_dir`

Correct other, minor documentation strings too.

### Issues Resolved ###

- closes #INTEG-1091

### Testing Procedure ###
* Call any of the methods (listed in CHANGELOG) and make sure you get a Py42Response object as a response. Before, it was just None.

### PR Checklist ###
Did you remember to do the below?

- [n/a] Add docstrings for any new public parameters / methods / classes
